src/main/java/com/mongodb/jdbc/MongoStatement.java

```diff
@@ -244,9 +251,16 @@ private ResultSet executeDirectClusterQuery(String sql)
             currentDB = conn.getDatabase(dbName);
         }
 
+        // Translate the SQL query
         BsonDocument catalogDoc =
                 mongoSQLTranslate.buildCatalogDocument(currentDB, dbName, namespaces);
+        logger.log(Level.FINE, "Namespaces schema: " + catalogDoc);
```

Comment:
Should we call this "catalog" in the logs instead of "namespaces schema"?

Reply:
I also like the term catalog since that's how we refer to it internally in mongosql

Reply:
I opted out of using "catalog" because I didn't want a mix up with the JDBC use of the term, which really means DBs names.